README.md

```diff
@@ -270,6 +270,28 @@ Configure ccache (installed by kpatch package):
 
     ccache --max-size=5G
 
+#### Gentoo
+
+*NOTE: You'll need about 15GB of free disk space for the kpatch-build cache in
+`~/.kpatch` and for ccache.*
+
+Install Kpatch and Kpatch dependencies:
+
+```bash
+emerge --ask sys-kernel/kpatch
+```
```

Comment:
This is a little different than the instructions for the other distros, which install build and runtime dependencies rather than the kpatch package itself.  (Please forgive my lack of Gentoo knowledge here.)  I suppose the net effect might be equivalent for a source-based distros like Gentoo, but with the added side effect of clean kpatch binaries in their installed paths.  Could that lead to confusion if the user decides to then build kpatch.git but not `make install` ?

Reply:
actually i'm not sure what the other distro are doing.
with emerge --ask sys-kernel/kpatch it will install all the needed dependency for having a working kpatch in the system that is recommended instead of installing it from source.
user also have a live package that can be enabled using the live flag for download and install the latest kpatch from github.

Reply:
The Installation section for the other distros could have simply said `yum/dnf/apt-get install kpatch` and let the package manager do all of the heavy lifting.  However, the way it's currently written, it is as if the user has a fairly minimal (or default) distro install and needs to build and use kpatch from these git sources.  IOW, I read this section of  `README.md` as a user-readable translation of a kpatch specfile.  I suspect it was originally written this way before distributions were providing kpatch packages -- maybe it's time we update to suggest the user installs official builds unless the they know what they're doing.

For the time being, I think the instructions are targeting developers looking to get started on kpatch.git development work on their given distro (without installing an official kpatch package).

Reply:
I think for Gentoo is still better if the user use emerge for installing Kpatch, for not breaking the distribution FHS.
Is maybe better writing both way ?